grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/SyntaxTreeGenerator.java

```diff
@@ -237,17 +271,11 @@ public static SyntaxTree generateSyntaxTreeForServiceSample(ServiceStub serviceS
         );
         service.addAnnotation(grpcServiceDescriptor.getAnnotationNode());
 
-        List<Method> methodList = new ArrayList<>();
-        methodList.addAll(serviceStub.getUnaryFunctions());
-        methodList.addAll(serviceStub.getClientStreamingFunctions());
-        methodList.addAll(serviceStub.getServerStreamingFunctions());
-        methodList.addAll(serviceStub.getBidiStreamingFunctions());
-
         for (Method method : methodList) {
             Function function = new Function(method.getMethodName());
             function.addQualifiers(new String[]{"remote"});
-            String input = method.getInputType();
-            String output = method.getOutputType();
+            String input = getType(method.getInputType());
```

Comment:
Instead of having a `getType` function, shall we handle it from the `stubFile` object? For instance, check the input/output type in the protobuf file and if it is `Timestamp` change the input/output type to `time:Utc`.

Reply:
Can add `time:Utc` as a new case inside [BalGenerationUtils](https://github.com/ballerina-platform/module-ballerina-grpc/blob/5cc02167df83a28518d0d499a63b03f12ff95866/grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/balgen/BalGenerationUtils.java#L81), but `method.getInputType(), method.getOutputType()` is used in some places to get method names/context names also. (eg - [UnaryUtils](https://github.com/ballerina-platform/module-ballerina-grpc/blob/5cc02167df83a28518d0d499a63b03f12ff95866/grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/utils/UnaryUtils.java#L139))

Reply:
Yes. We can have the input/output types as `time:Utc` and construct method/context names accordingly. For eg:
`byte[]` -> `ContextBytes` similarly,
`time:Utc` -> `ContextTimeStamp`
Check https://github.com/ballerina-platform/module-ballerina-grpc/blob/5cc02167df83a28518d0d499a63b03f12ff95866/grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/utils/UnaryUtils.java#L136

Reply:
changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/7542e0e21e09589aa10e28ae97f881c3ccc4d7a2